knowledge-base/drawer-sync-selected-item.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How to select drawer item if menu page is opened by using its URL
+description: How to select the correct drawer item when you don't navigate with the drawer
+type: troubleshooting
+page_title: Sync Drawer item selection with external navigation
+slug: drawer-kb-sync-selected-item
+position: 
+tags: 
+ticketid: 1530772
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Drawer for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
```

Comment:
Copy-pasting the ticket contents as is does not necessarily guarantee capturing the best possible customer context. Consider fixing the customer's grammar and spelling, and improve SEO with applicable keywords.

Reply:
Absolutely.